doc/users/whats_new/rcparams.rst

```diff
@@ -1,3 +1,9 @@
+Added ``xtick.minor.visible`` and ``ytick.minor.visible`` key to rcParams
+`````````````
+Two new keys to control the minor ticks on x/y axis respectively, default set to False(hide minor ticks on the axis).
+
+When enabled, it will present the automatic minor ticks.
```

Comment:
"When ``True`` the minor ticks are shown and located via a``mticker.AutoMinorLocator()``."